### ISSUE TITLE
Consider changing default branch from master to something more appropriate

### ISSUE THREAD

**Issue by jamincollins** [archlinux/archinstall#1125]
Reference:
* https://sfconservancy.org/news/2020/jun/23/gitbranchname/

**Comment by Torxed** [comment]
Apologies if you feel the title change didn't have the same oumph as your original headline. But it felt a bit less professional and more something that draws attention towards the title *(and not the topic)*. I also felt it was a bit of a insensitive stab at me as a person as I believe the context of every subject matters. And I have no history of being insensitive *(I hope)* but rather I've been pretty welcoming to all changes, all people and I've made it a priority to make everyone feel heard and their opinions valued even if I don't fully agree every time.

To make that even more clear, I want to point out that it was the default naming convention and best practice at the time of creating this repository and I really didn't think changing the name was in proportion between what benefit it would do vs the headaches it would cause on a technical level. We are all here as developers, because we have a passion for the technical interest, meaning we're equal and belong no matter preferences or phrasing while we work to improve archinstall. That said It's been many years and i'm open for renaming it. But in the context where this topic is discussed in - it has no affiliation or meaning to reflect a culture of any kind but rather the adjective. I also believe quite strongly in freedom of speech and the power of diversity *(not only people but words diversity as well)*. I'm pretty sure no one associates a github repo with the historical mistreatments that you refer to. In which case we would change the name simply to make a statement of support for those oppressed cultures that the article implies.

I do not know how much influence archinstall have in the matter, if we have a lot i can prioritize this - otherwise I'll prioritize finishing the current work already in the pipeline and then revisit the renaming (and all the communication and update it entails).

**Comment by jamincollins** [comment]
> I also felt it was a bit of a insensitive stab at me as a person as I believe the context of every subject matters. 

My apologies, it wasn't intended as a stab at anyone.  Simply raising an issue that I stumbled upon.  I didn't notice the default branch until I started working on potential contributions to the project.

Some of the points you've raised are more or less true for the vast majority of the industry.  For example "it was the default naming convention" and "in the context where this topic is discussed in - it has no affiliation or meaning to reflect a culture of any kind but rather the adjective". Both of these are true of most git repositories.  Yet groups, companies, and organizations made a change, here are a few:
* https://github.blog/changelog/2020-10-01-the-default-branch-for-newly-created-repositories-is-now-main/
* https://about.gitlab.com/blog/2021/03/10/new-git-default-branch-name/
